### PR TITLE
Added example how to use JsonUnit

### PR DESCRIPTION
**Pull request by lukas-krecan**
Added example how to use JsonUnit.

### CODE REVIEW

**Comment by lukas-krecan** [comment]
I am not sure if I am supposed to add the example to Spring test or what's expected. Feel free to send suggenstions.

File: framework-docs/modules/ROOT/pages/testing/mockmvc/assertj/assertions.adoc [hunk 1]
```diff
@@ -47,3 +47,16 @@ include-code::./FamilyControllerTests[tag=assert-file,indent=0]
 
 If you prefer to use another library, you can provide an implementation of
 {spring-framework-api}/test/json/JsonComparator.html[`JsonComparator`].
+
+Alternatively, you can use AssertJ https://github.com/assertj/assertj/blob/3.x/assertj-core/src/main/java/org/assertj/core/api/AssertFactory.java[AssertFactory].
```
Comment (snicoll) [inline framework-docs/modules/ROOT/pages/testing/mockmvc/assertj/assertions.adoc:51]: Please rephrase that to be consistent with previous sentences. For instance.

JsonUnit is also supported via the standard `convertTo`method. I am not keen to start referring to AssertJ's Javadoc for this.

File: framework-docs/modules/ROOT/pages/testing/mockmvc/assertj/assertions.adoc [hunk 2]
```diff
@@ -47,3 +47,16 @@ include-code::./FamilyControllerTests[tag=assert-file,indent=0]
...
+For example, if you want to use https://github.com/lukas-krecan/JsonUnit[JsonUnit] you can
+do this
+
+```
+import static net.javacrumbs.jsonunit.assertj.JsonAssertions.jsonUnitJson;
```
Comment (snicoll) [inline framework-docs/modules/ROOT/pages/testing/mockmvc/assertj/assertions.adoc:56]: This isn't markdown. We don't compile against JSonUnit so we can't add the example.

**Review by snicoll — CHANGES_REQUESTED** [review]
I've added a couple of comments.

**Comment by snicoll** [comment]
Closing due to the lack of feedback.

**Pull request closed without merging**